pkg/controllers/cassandra/nodepool/resource.go

```diff
@@ -195,10 +179,6 @@ func StatefulSetForCluster(
 									Name:  "CASSANDRA_RACK",
 									Value: "Rack1-K8Demo",
 								},
-								{
-									Name:  "CASSANDRA_AUTO_BOOTSTRAP",
-									Value: "false",
-								},
```

Comment:
The Docker image doesn't pay attention to this environment variable so I removed it.

We'll see if it causes any dataloss problems when I rebase #225